randovania/games/prime1/gui/preset_settings/prime_patches_qol.py

```diff
@@ -60,7 +84,11 @@ def persist(value: bool):
 
     def _on_cutscene_changed(self, value: LayoutCutsceneMode):
         with self._editor as editor:
-            editor.set_configuration_field("qol_cutscenes", value)
+            try:
+                editor.set_configuration_field("qol_cutscenes", value)
+            except Exception:
```

Comment:
this should ideally be catching the specific relevant type of Exception rather than Exception itself